include/odp/api/spec/schedule.h

```diff
@@ -187,6 +235,42 @@ void odp_schedule_prefetch(int num);
  */
 int odp_schedule_num_prio(void);
 
+/**
+ * Initialize schedule configuration options
+ *
+ * Initialize an odp_schedule_config_t to its default values.
+ *
+ * @param[out] config  Pointer to schedule configuration structure
+ */
+void odp_schedule_config_init(odp_schedule_config_t *config);
+
+
+/**
+ * Global schedule configuration
+ *
+ * Initialize and configure scheduler with global configuration options.
+ *
+ * @param config   Pointer to scheduler configuration sturcture
+ *
+ * @retval 0 on success
+ * @retval <0 on failure
+ *
+ * @see odp_schedule_capability(), odp_schedule_config_init()
+ */
+int odp_schedule_config(const odp_schedule_config_t *config);
```

Comment:
It looks like implementing `odp_schedule_config()` requires that we also add `odp_schedule_start()`.

It should be stated when it's OK to call `odp_schedule_config()` and if this is optional. A proper startup sequence should also be defined.
E.g.
```
odp_schedule_config_init()
odp_schedule_config()
odp_schedule_start()
odp_schedule()
```

Reply:
Also when packet input/crypto/ipsec/timer send events to scheduler, the config/start order needs to be defined against those as well.

Maybe simply, odp_schedule_start() must be called before application itself or any ODP module creates any (scheduled)  queues.

Reply:
Yes. We need to add odp_schedule_start() and I believe schedule start should have priority before odp_pktio_start().